buzzmobile/sense/inputer/inputer.lisp

```diff
@@ -0,0 +1,7 @@
+(in-package :buzzmobile)
+
+(defun inputer ()
+    (with-ros-node ("inputer")
+        (let ((i 0) (pub (advertise "destination" "std_msgs/String")))
+            (loop-at-most-every .1
```

Comment:
How was this this interval chosen? should it be factored out and made available to more nodes?